runtime/callback.c

```diff
@@ -47,13 +66,11 @@ static void init_callback_code(void)
 
 CAMLexport value caml_callbackN_exn(value closure, int narg, value args[])
 {
-  CAMLparam0();
-  struct stack_info* parent_stack;
+  CAMLparam1(closure);
+  CAMLxparamN(args, narg);
+  CAMLlocal2(res, cont);
```

Comment:
`res` need not be a `CAMLlocal` since there is no allocation after `res` is initialised.